src/hazelweb/gui/UndoHistoryPanel.re

```diff
@@ -258,6 +258,7 @@ let view = (~inject: ModelAction.t => Vdom.Event.t, model: Model.t) => {
     | Line(_, line_content) =>
       switch (line_content) {
       | EmptyLine => indicate_words_view("empty line")
+      | CommentLine(_) => indicate_words_view("comment")
```

Comment:
@PPER any thoughts here on how comment lines should appear in undo panel? Here is what it looks like now
<img width="1638" alt="image" src="https://user-images.githubusercontent.com/8718124/87466208-df21be00-c5e3-11ea-9dc6-9249c409e05c.png">